.github/workflows/lock.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: '30'
+          issue-exclude-labels: 'Discussion'
+          issue-lock-comment: >
+            This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
+            
+            Please note this issue tracker is not a help forum. We recommend using [StackOverflow](https://stackoverflow.com/questions/tagged/jestjs) or our [discord channel](https://discord.gg/j6FKKQQrW9) for questions.
```

Comment:
I just added this as-is. Not sure if it should be "usage questions" or something? Also, if the docs for some feature/bug fix are bad, that's perfectly fine to open a new issue about, I don't want to discourage that